packages/turbo-workspaces/src/getWorkspaceDetails.ts

```diff
@@ -1,3 +1,4 @@
+import { Workspace } from "@turbo/repository";
```

Comment:
- I don't see this being used, is it a mistaken import?
- was this already in turbo-workspaces package.json?
- If not, `@turbo/repository` has native dependencies, so anyone using `@turbo/workspaces` will now get hit with a "supported platforms" issue if they aren't one one.

Reply:
Ah yeah that made it in by mistake